vnda/loaders/productListingPage.ts

```diff
@@ -101,7 +104,7 @@ const searchLoader = async (
   const sort = url.searchParams.get("sort") as Sort;
   const page = Number(url.searchParams.get("page")) || 1;
 
-  const isSearchPage = url.pathname === "/busca";
+  const isSearchPage = url.pathname === "/busca" || url.pathname === "/s" || url.pathname === props.searchPagePath;
```

Comment:
If url.pathname is undefined, ok verify /busca and /s. But if it is filled, we should look only to thath
```suggestion
  const isSearchPage = props.searchPagePath ? props.searchPagePath ===  url.pathname : url.pathname === "/busca" || url.pathname === "/s"```